robot/client/client_test.go

```diff
@@ -121,17 +120,17 @@ func TestStatusClient(t *testing.T) {
 	}
 
 	// TODO(RSDK-882): will update this so that this is not necessary
```

Comment:
RSDK-882 is closed, what's the status of this `TODO`?

Reply:
I'm not sure.  Theres 7 instances of this TODO in tests.  I'm going to try to remove the offending code and see if it breaks now.

Reply:
Nice catch! This was made totally unnecessary by recent frame system changes.  Removed.